modules/templates.js

```diff
@@ -0,0 +1,13 @@
+angular.module('pages').run(['$templateCache', function($templateCache) {
```

Comment:
This file, `modules/templates.js`, should instead be `modules/pages/scripts/templates.js`